ops/testing.py

```diff
@@ -85,6 +87,7 @@ def __init__(
         self._oci_resources = {}
         self._framework = framework.Framework(
             self._storage, self._charm_dir, self._meta, self._model)
+        self.update_config(key_values=self._load_config_defaults(config))
```

Comment:
I wonder if we want a common function that wouldn't potentially cause us to trigger events due to a config update.
I realize that in this case it doesn't, because begin() hasn't been called, but it does feel like that is luck not design. So maybe pulling out the common bits into an _update_config that doesn't have side effects?

Reply:
Done.